sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVFileFormat.scala

```diff
@@ -101,6 +101,12 @@ class CSVFileFormat extends TextBasedFileFormat with DataSourceRegister {
     val broadcastedHadoopConf =
       sparkSession.sparkContext.broadcast(new SerializableConfiguration(hadoopConf))
 
+    val columnNameOfCorruptRecord = csvOptions.columnNameOfCorruptRecord.getOrElse(sparkSession
+      .sessionState.conf.columnNameOfCorruptRecord)
```

Comment:
Maybe, we could put `columnNameOfCorruptRecord` as an argument in `CSVOptions` consistently with `JSONOptions`. I guess then it won't require the changes between 129L - 133L as well.